src/MarketingPageAcceptanceTestsSpecflow/Steps/AboutSolution/Suppliers_EditSupplierAssertedIntegrationsSection.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using MarketingPageAcceptanceTestsSpecflow.Utils;
+using TechTalk.SpecFlow;
+
+namespace MarketingPageAcceptanceTestsSpecflow.Steps.AboutSolution
+{
+    [Binding]
+    public class Suppliers_EditSupplierAssertedIntegrationsSection : TestBase
+    {
+        public Suppliers_EditSupplierAssertedIntegrationsSection(UITest test, ScenarioContext context) : base(test, context)
+        {
+
+        }
+
+        [Given(@"the User has entered (\d{3,4}) characters on the Catalogue Solution integrations page in the (Integrations) section")]
+        public void GivenTheUserHasEnteredCharactersOnTheCatalogueSolutionIntegrationsPageInTheIntegrationsSection(int characters, string section)
+        {
+            _test.pages.Dashboard.NavigateToSection(section);
+            _test.pages.SolutionDescription.LinkAddText(characters);
+        }
+        
+        [Given(@"the Integrations section does not require Mandatory Data")]
```

Comment:
I have a feeling we have a few steps that are very similar that could be refactored to meet this step. Might be worth taking a look at some point